README.md

```diff
@@ -141,9 +141,9 @@ This monorepo is comprised of the following packages, products, and examples:
 
    ```bash
    # Compile and deploy contracts
-   cd packages/contracts
-   pnpm build
+   pnpm --dir packages/contracts build
    pnpm --dir packages/contracts run publish ../sdk/src/abi
```

Comment:
For some reason this is causing something to break when the `/abi` folder changes. GuardianRecoveryModuleAbi and SsoAccountAbi are removed from the `index.ts` inside the /abi folder.